ckanext/unaids/theme/templates/package/releases/list.html

```diff
@@ -0,0 +1,110 @@
+{% extends "package/read_base.html" %}
+
+{% block primary_content_inner %}
+
+<div class="btn-group">
+    <a href="{{ h.url_for('unaids_dataset_releases.release_view', dataset_type=dataset_type, id=pkg.id) }}" class="btn btn-danger">Add a new Release</a>
+    <button type="button" class="btn btn-danger dropdown-toggle" data-toggle="dropdown">
+        <span class="caret"></span>
+    </button>
+    <ul class="dropdown-menu">
+        <li><a href="{{ h.url_for('unaids_dataset_releases.release_view', dataset_type=dataset_type, id=pkg.id) }}">From the <u>latest version</u> of the dataset</a></li>
+        <li><a href="{{ h.url_for('dataset.activity', id=pkg.id, create_release=True) }}">From a <u>past version</u> of the dataset</a></li>
+    </ul>
+</div>
+
+<br /><br />
+
+{% if dataset_releases %}
+    <link rel="stylesheet" href="/bootstrap-table.css" />
+    <ul id="ReleasesLoading" class="list-group">
+        <li class="list-group-item text-center text-muted">
+            <h2><i class="fa fa-spinner fa-spin"></i> Loading</h2>
+        </li>
+    </ul>
+    <div id="ReleasesTableContainer">
+        <table>
+            <thead>
+                <tr>
+                    <th data-width="50"></th>
+                    <th></th>
+                    <th class="small-column text-right"></th>
+                    <th class="small-column text-right"></th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for release in dataset_releases %}
+                <tr>
+                    <td class="text-right vertical-align-top">
+                        <h4><i class="fa fa-tag text-muted"></i></h4>
+                    </td>
+                    <td class="vertical-align-top">
+                        <h4><a href="{{ h.url_for('dataset.read', id=pkg_dict.id) }}">{{ release.name }}</a></h4> 
+                        <div class="text-muted">{{ release.notes }}</div>
+                        <div class="text-muted">Created by <a href="#">{{ release.author }}</a></div>
+                        <br>
+                    </td>
+                    <td>{{ release.created }}</td>
```

Comment:
`release.created` will probably be returning UTC time in format: `YYYY-MM-DD HH:MM:SS`
There's a helper you could use here:
```
#ckan/ckan/templates/snippets/activities/new_resource.html:11
    <span class="date">
      {{ h.time_ago_from_timestamp(activity.timestamp) }}
    </span>
```